images/linux/scripts/installers/nodejs.sh

```diff
@@ -9,6 +9,16 @@ curl -sL https://raw.githubusercontent.com/mklement0/n-install/stable/bin/n-inst
 ~/n/bin/n lts
 npm install -g grunt gulp n parcel-bundler typescript newman
 npm install -g --save-dev webpack webpack-cli
+
+# Install the Netlify CLI
+npm i -g netlify-cli --unsafe-perm=true
```

Comment:
let's add a comment why we need this `--unsafe-perm=true` flag

Reply:
Thank you, added comment